cctrl/auth.py

```diff
@@ -92,12 +132,73 @@ def delete_tokenfile(settings):
     return False
 
 
+def set_configfile(settings, email=None, ssh_auth=None, ssh_path=None):
+    create_config_dir(settings)
+    config = ConfigParser.ConfigParser()
+    config.read(settings.config_path)
+
+    if not config.has_section('user'):
+        config.add_section('user')
+
+    if email:
+        config.set('user', 'email', email)
+
+    if ssh_auth is not None:
+        config.set('user', 'ssh_auth', ssh_auth)
+
+    if ssh_path:
+        config.set('user', 'ssh_path', ssh_path)
+
+    with open(settings.config_path, 'w') as configfile:
+        config.write(configfile)
+
+    return True
+
+
+def get_configfile(settings):
```

Comment:
I can rename

`set_configfile` -> `set_user_config`
`get_configfile` -> `get_user_config`

The main motivation was not get confused with the `config` command calling these methods `set_config` or `get_config`.